services/manifest-repo-export-service/pkg/service/service.go

```diff
@@ -0,0 +1,47 @@
+/*This file is part of kuberpult.
+
+Kuberpult is free software: you can redistribute it and/or modify
+it under the terms of the Expat(MIT) License as published by
+the Free Software Foundation.
+
+Kuberpult is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+MIT License for more details.
+
+You should have received a copy of the MIT License
+along with kuberpult. If not, see <https://directory.fsf.org/wiki/License:Expat>.
+
+Copyright freiheit.com*/
+
+package service
+
+import (
+	"fmt"
+	"net/http"
+
+	xpath "github.com/freiheit-com/kuberpult/pkg/path"
+	"github.com/freiheit-com/kuberpult/services/manifest-repo-export-service/pkg/repository"
+)
+
+type Service struct {
+	Repository repository.Repository
+}
+
+func (s *Service) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	head, _ := xpath.Shift(r.URL.Path)
+	switch head {
+	case "health":
+		s.ServeHTTPHealth(w, r)
+	case "release":
+		w.WriteHeader(http.StatusNotFound)
```

Comment:
we don't need to reimplment this special "release" endpoint.
However, there should be a default for the `switch`